README.md

```diff
@@ -250,7 +278,23 @@ will take precedence.
 
 Hyrax is available under [the Apache 2.0 license](LICENSE.md).
 
-# Contributing
+# Quick Start for Hyrax development and contributing code back
```

Comment:
@mjgiarlo Will the dev guide move to projecthydra.github.io?  Seems like that is the best place for it going forward.  @afred @jenlindner @bess